### ISSUE TITLE
v1.5.10 [beta]

### ISSUE THREAD

**Issue by kleuter** [crystalidea/macs-fan-control#511]
What's new

- Fixed Not showing SMC sensors on Hackintosh  #448

[macsfancontrol_beta.zip](https://github.com/crystalidea/macs-fan-control/files/6531793/macsfancontrol_beta.zip)

**Comment by Vorshim92** [comment]
it crashes when I close it (when it goes to the tray icon) :( but it works perfectly!!!

**Comment by kleuter** [comment]
Please post the crash report.

**Comment by Vorshim92** [comment]
> Please post the crash report:
> https://bjango.com/help/crashlogmac/

here you are and thanks!  (of course it's an hackintosh xD) 
[Macs Fan Control_2021-05-27-143911_MacBook-Pro-di-Stefano-2.crash.zip](https://github.com/crystalidea/macs-fan-control/files/6554022/Macs.Fan.Control_2021-05-27-143911_MacBook-Pro-di-Stefano-2.crash.zip)

**Comment by kleuter** [comment]
Please post a screenshot of the app as well

**Comment by kleuter** [comment]
If possible, please record a video of the crash. Thank you

**Comment by Vorshim92** [comment]
> If possible, please record a video of the crash. Thank you

ok! Video incoming. I found that the trigger who makes the app crash is the "automatic/personal" setting of the control tab that is missing at start. but if you set the automatic preset it appears again and it works perfectly but you have to do this everytime. So i think is that 


https://user-images.githubusercontent.com/7747380/119842780-43e9bc80-bf07-11eb-9da8-c54547dd86d1.MP4

**Comment by Vorshim92** [comment]
> Thanks. Please copy tech info from the More menu as well

[moreinfo.txt](https://github.com/crystalidea/macs-fan-control/files/6559453/moreinfo.txt)

here u are!

**Comment by kleuter** [comment]
Will fix it soon, thanks!

**Comment by Vorshim92** [comment]
> Will fix it soon, thanks!

I'm on 11.5 beta but if you want I can send you log and everything from 11.2.3 and 11.4. I have another dell xps with the same problem so I don't think it's the beta. Thanks to you man, really! :D

> Will fix it soon, thanks!

**Comment by kleuter** [comment]
Please download the latest beta from the link in the first post.

**Comment by Vorshim92** [comment]
> Please download the latest beta from the link in the first post.

thanks so much! it works perfectly! thanks thanks thanks thanks thanks thanks thanks

**Comment by Lorys89** [comment]
@kleuter when is the official release scheduled?

**Comment by kleuter** [comment]
next month

**Comment by Lorys89** [comment]
> What's new:
> 
> * Fixed Not showing SMC sensors on Hackintosh  #448
> * Fixed app stopped responding after macOS Big Sur upgrade (Apple Silicon) #500
> 
> [Download for macOS 10.13+](https://crystalidea.com/downloads/macsfancontrol_dev.tar.xz)

@kleuter not work the app 
![Schermata 2021-05-29 alle 14 06 38](https://user-images.githubusercontent.com/64613690/120069613-20e61680-c087-11eb-8ec4-c2d189037cdc.png)

**Comment by i0n1ca** [comment]
![Screenshot 2021-06-14 at 17 37 17](https://user-images.githubusercontent.com/3650133/121910641-b1834e80-cd37-11eb-8cc6-b7bf886575b8.png)

The app is running, temperature is showing but this pop-up shows on every time I open the app. The Administrator password was entered but no success.

**Comment by kleuter** [comment]
Did you have Macs Fan Control installed before?

**Comment by i0n1ca** [comment]
Yes, I had version 1.5.9 beta from the start of the page here on github. Now I have replaced it with version beta 10.

**Comment by i0n1ca** [comment]
No such folder. 
<img width="444" alt="Screenshot 2021-06-14 at 18 16 13" src="https://user-images.githubusercontent.com/3650133/121916231-a121a280-cd3c-11eb-9937-fee0964ecd32.png">

**Comment by kleuter** [comment]
Please execute the following command in Terminal and start the app again:
```
sudo mkdir /Library/PrivilegedHelperTools
```

**Comment by i0n1ca** [comment]
![Screenshot 2021-06-14 at 18 20 48](https://user-images.githubusercontent.com/3650133/121916923-4177c700-cd3d-11eb-80ff-30efddf7b3d7.png)
Strange, in terminal the folder exists and cannot be created.

**Comment by kleuter** [comment]
This folder exists, what files are inside?

**Comment by i0n1ca** [comment]
![Screenshot 2021-06-14 at 18 22 09](https://user-images.githubusercontent.com/3650133/121917103-708e3880-cd3d-11eb-8911-0c3f4b95c7b2.png)
These files.

**Comment by kleuter** [comment]
Use **ls -l** , please (to see file size)

**Comment by kleuter** [comment]
Please post tech info from the More menu as well in Macs Fan Control

**Comment by i0n1ca** [comment]
![Screenshot 2021-06-14 at 18 25 12](https://user-images.githubusercontent.com/3650133/121917518-ded2fb00-cd3d-11eb-87bb-b677aea2db82.png)
[technical info.docx](https://github.com/crystalidea/macs-fan-control/files/6649516/technical.info.docx)

**Comment by i0n1ca** [comment]
I just quit the app and started again without any pop-up after password screen. It seems that is ok now.

**Comment by kleuter** [comment]
Thanks for the info, so you saw the warning message only once?

**Comment by i0n1ca** [comment]
I had tried twice and the message pop-up twice. Now was the third and it dissapeared.

**Comment by kleuter** [comment]
No problem, Thanks for the information, it will help if the issue will happen to other users!

**Comment by kleuter** [comment]
Build 34 released

**Comment by kleuter** [comment]
Build 35 released

**Comment by Kimsdin** [comment]
I bought a license for Macs Fan Control Pro for my Imac 27 "with T2 Chip. 
However, I'm not exactly PT. particularly impressed as I somehow got a version of Macs Fan Control which does not work optimally it was a ver 1.5.10 so therefore uninstalled it and would have then downloaded your version 1.5.9 but it will not run on my Mac since I the earlier version was 1.5.10. now I can not find that version 1.5.10 again, So therefore my question is so how do I get version 1.5.9 to work on my Imac ????

**Comment by Kimsdin** [comment]
Help I cant install v1.5.9 after deleting 1.5.10 beta i get this message 
<img width="314" alt="Skærmbillede 2021-09-26 kl  17 17 21" src="https://user-images.githubusercontent.com/91417304/134813966-32d93009-45ee-400d-92c2-afed2d06db24.png">

**Comment by kleuter** [comment]
Please run v1.5.10 and delete it completely via menu More -> Uninstall

**Comment by kleuter** [comment]
Build 36 released, most likely the last one before the release
